src/app/public/modules/datepicker/datepicker-input-fuzzy.directive.ts

```diff
@@ -409,7 +416,7 @@ export class SkyFuzzyDatepickerInputDirective
   public onInputBlur(): void {
     this.onTouched();
 
-    let formattedDate = this.fuzzyDateService.getStringFromFuzzyDate(this.value, this.dateFormat);
+    let formattedDate = this.fuzzyDateService.format(this.value, this.dateFormat, this.locale);
```

Comment:
Kind of talking out loud here - Do we even need `getStringFromFuzzyDate` anymore? It may be one of those functions used by our consumers. What this PR reveals is that method wasn't considering locale at all... probably a bad thing. Is this a gap?

Reply:
Lets discuss in flex tomorrow. We could potentially deprecate the method. But there are 2 SPAs that currently use it.